master/getting-started/kubernetes/host-endpoints.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Using Host Endpoints with Kubernetes
+---
+
+When [Using Calico to Secure Host Interfaces]({{site.baseurl}}/{{page.version}}/getting-started/bare-metal/bare-metal) 
+by setting up [Host Endpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/hostendpoint) 
+there are some additional considerations needed when using Kubernetes.
+
+### Allowing forwarded traffic in Host Endpoint Policy
+
+Since version v2.1.0 of Calico, Host Endpoint(HEP) policies apply to forwarded
+traffic.  Due to the forwarding that kubernetes sets up through kube-proxy,
+to handle its service IPs and NodePorts, it is necessary to add policy to allow
```

Comment:
I think the emphasis is wrong here.  It is not necessary to use host endpoints at all.  You can use host endpoints to restrict access to your cluster.  If you want to restrict access to only allow kubernetes node ports then you can use the following policy.  I would encourage the whole doc be oriented in this kind of +ve context.  Emphasis on what  the user is trying to achieve.